### ISSUE TITLE
Usage of link rel=author in head.html

### ISSUE THREAD

**Issue by anniegreens** [MattSLangford/Tiny-Theme-for-Micro.blog#12]
In customizing a `layouts/partials/head.html` I discovered that there is a `<link>` tag using `rel="author"` pointing to a `humans.txt` with Matt's info. I wrote up my concerns with this in a microblog post and why I don't think it is the proper usage of the attribute: https://micro.anniegreens.lol/2024/06/19/i-hope-i.html

Perhaps this is an oversight and was part of your own theme and wasn't removed prior to contributing to this repo. I went ahead and removed the reference for my own theme, which I've done quite a bit of customization on, but for other folks using Tiny Theme this is telling the machines that the author of the current page or article is who is in that `humans.txt` and that seems incorrect according to my understanding of the attribute, MDN, and even per the Microformats Wiki: http://microformats.org/wiki/rel-author

As I noted in my microblog post, I am extremely grateful for Tiny Theme and I do reveal attribution in the footer, but I think the rel attribute is misplaced. I hope this is the right place to file this, I thought about putting it in the Micro.blog help center but it didn't feel relevant there this time.

Thanks!

**Comment by MattSLangford** [comment]
I understand your point, but let me offer some clarification. Humans.txt is an expansion on the basic `rel="author"`, created in an effort to allow attribution for designers/developers (or a complete team) behind the site.

In their own words, humans.txt is "an initiative for knowing the people behind a website. It's a TXT file that contains information about the different people who have contributed to building the website."

[Here is a link for more information](https://humanstxt.org).

**Comment by AeliusSaionji** [comment]
The humans.txt wording is somewhat ambiguous on this subject, but the intent is clearly focused on being analogous to robots.txt.
The use of the html author tag is stated as a suggestion for added discovery, presumably in scenarios where the site owner and document author are one. I do not see any text that supports the idea that the intent is to expand upon or alter the tag's intended use.

See also these snippets from the humanstxt site, emphasis mine:
> If possible, you **can also** add an author tag


> Humans.txt is just a way to have more information about the **authors of the _site_**.

Following the robots.txt convention, humans.txt would provide attribution to the contents of the domain domain or subdomain in which it lives. (actually it's even narrower in scope, but I digress.)
Conversely, the html author tag is intended to be per document, and it should attribute the document's featured content.

If you author a document under a (sub)domain you control, then it makes sense to direct traffic to the humans.txt- which should exist within the same (sub)domain as the document.

You could state somewhere on your page that you wish to be listed under your users' human.txt. In the same vein as robots.txt, compliance would be entirely voluntary.

It's an interesting idea- you may want to link to the humanstxt site in your humans.txt, so people can read about the idea behind the file they've found.

**Comment by MattSLangford** [comment]
I believe humans.txt was created for a scenario where the site owner and **site author** are NOT one. In fact, their home page differentiates between the two multiple times.

> More often than not, the owners of the site don't like the authors signing it;

In this context, I believe their definition of authors is not referring to _content writers_ but is intentionally referencing **designers and developers**, amongst others. 

There is a clear conflict between the original intention of `rel=authors` and the goal of `humans.txt`. The latter is essentially hijacking the former for the benefit of those "people who may take part of the creation of a site." Furthermore, it is meant as a sitewide function, not a per-document feature…another conflict between the two.

Having said all of that, there is no benefit or cost in having a `humans.txt` listed as `rel="author"`. The progression of `humans.txt` is stagnant, seeing no meaningful movement in the last decade. It is essentially an antiquated way for people like us to dive into certain site specifics. For that reason, I removed the line from the head in version 8.2.1.

For now, I am keeping `humans.txt`, but I've added clarifying verbiage on the page for anyone who stumbles across it. I'll also further document it for theme users.
